preferencesfx/src/main/java/com/dlsc/preferencesfx/util/StorageHandler.java

```diff
@@ -1,243 +1,145 @@
 package com.dlsc.preferencesfx.util;
 
-import static com.dlsc.preferencesfx.util.Constants.DEFAULT_DIVIDER_POSITION;
-import static com.dlsc.preferencesfx.util.Constants.DEFAULT_PREFERENCES_HEIGHT;
-import static com.dlsc.preferencesfx.util.Constants.DEFAULT_PREFERENCES_POS_X;
-import static com.dlsc.preferencesfx.util.Constants.DEFAULT_PREFERENCES_POS_Y;
-import static com.dlsc.preferencesfx.util.Constants.DEFAULT_PREFERENCES_WIDTH;
-import static com.dlsc.preferencesfx.util.Constants.DIVIDER_POSITION;
-import static com.dlsc.preferencesfx.util.Constants.SELECTED_CATEGORY;
-import static com.dlsc.preferencesfx.util.Constants.WINDOW_HEIGHT;
-import static com.dlsc.preferencesfx.util.Constants.WINDOW_POS_X;
-import static com.dlsc.preferencesfx.util.Constants.WINDOW_POS_Y;
-import static com.dlsc.preferencesfx.util.Constants.WINDOW_WIDTH;
-
 import com.dlsc.preferencesfx.model.Setting;
-import com.google.gson.Gson;
-import java.security.MessageDigest;
-import java.security.NoSuchAlgorithmException;
-import java.util.ArrayList;
-import java.util.Objects;
-import java.util.prefs.BackingStoreException;
-import java.util.prefs.Preferences;
-import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
+import java.util.prefs.Preferences;
 
 /**
- * Handles everything related to storing values of {@link Setting} using {@link Preferences}.
+ * Abstraction for storing values of {@link Setting}.
+ * Clients able to provide custom implementations by extending this class.
```

Comment:
We used the terminology "API user" throughout the documentation instead of "Clients" in this regard, I think it would make more sense to have it that way here as well
Also, it should be "implementing" this class, since interfaces can only be implemented, not extended